src/app/services/search/search.service.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { SearchService } from './search.service';
+import { environment } from 'src/environments/environment';
+
+describe('SearchService', () => {
+  let service: SearchService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [SearchService],
+      imports: [HttpClientTestingModule]
+    });
+    service = TestBed.get(SearchService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should use search service', () => {
+    service.searchProperties(`${environment.api_url}/properties/?`);
```

Comment:
Hey @JamesMudidi, we should endeavor to add assertions to test methods to verify the behavior we expect.